examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -788,10 +804,22 @@ def _is_episode_finished(self) -> bool:
 
     def _is_episode_successful(self) -> bool:
         """
-        Returns true if all agents finished the episode successfully.
+        Returns true if:
+        * Task success is >99%.
+        * All agents finished the episode without reporting an error.
         """
-        return all(
+
+        task_percent_complete = self._metrics.get_task_percent_complete()
+        task_successful = (
```

Comment:
What is the real situation for `rearrange_v2` that this metric is None? How about add a comment explaining why sometimes that metric is missing and why we want to consider the task successful in that case.